wire-ios/Wire-iOS Tests/UserStatusViewSnapshotTests.swift

```diff
@@ -130,13 +130,19 @@ final class UserStatusViewSnapshotTests: ZMSnapshotTestCase {
         updateAvailability(for: user, newValue: availability)
 
         let sut = UserStatusView(
-            user: user,
             options: options,
             userSession: userSession
         )
         sut.overrideUserInterfaceStyle = userInterfaceStyle
-        sut.backgroundColor = userInterfaceStyle == .dark ? .black : .white
+        sut.backgroundColor = .systemBackground
```

Comment:
I see snapshots for only light mode here. May be we should do the test for dark mode as well?

Reply:
Sorry, the [updated reference image](https://github.com/wireapp/wire-ios/blob/090f9159f8adfdb6fa4e7569525cf2799538bc51/wire-ios/Wire-iOS%20Tests/ReferenceImages/ConversationCellBurstTimestampViewSnapshotTests/testForIncludeDayOfWeekAndDot.1.png) doesn't actually belong to this test.

This test (`UserStatusViewSnapshotTests`) does have dark and light reference images: https://github.com/wireapp/wire-ios/tree/develop/wire-ios/Wire-iOS%20Tests/ReferenceImages/UserStatusViewSnapshotTests